bundle/internal/schema/annotations_openapi_overrides.yml

```diff
@@ -230,6 +230,10 @@ github.com/databricks/cli/bundle/config/resources.Pipeline:
   "run_as":
     "description": |-
       PLACEHOLDER
+  "target":
+    "deprecation_message": This legacy field is deprecated for pipeline creation in favor of the schema field
```

Comment:
All pipelines out there use this property and the "schema" field is not yet GA.

We should not mark it deprecated until we are sure we can suggest folks rename/update without migration pains.